charts/k8ez/templates/_helpers.tpl

```diff
@@ -12,16 +12,16 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
 If release name contains chart name it will be used as a full name.
 */}}
 {{- define "chart.fullname" -}}
-{{- if .Values.fullnameOverride -}}
-{{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" -}}
-{{- else -}}
-{{- $name := default .Chart.Name .Values.nameOverride -}}
-{{- if contains $name .Release.Name -}}
-{{- .Release.Name | trunc 63 | trimSuffix "-" | lower -}}
-{{- else -}}
-{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" | lower -}}
-{{- end -}}
-{{- end -}}
+  {{- if .Values.fullNameOverride -}}
```

Comment:
Why are we renaming this?

Reply:
Lets not rename this - it will break backwards compatibility and isn't a substantial enough improvement, I think.